### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by artemfastdev**
Fix radio-button z-index; change styled-compoennts to react-emotion in InsuranceTableRows;

### CODE REVIEW

File: src/components/chat/insurance-table-rows/InsuranceTableRows.tsx [hunk 1]
```diff
@@ -3,44 +3,48 @@ import { ACTIVATION_DATE, CANCELLATION_DATE } from 'lib/messageTypes'
...
-  align-items: center;
-  justify-content: center;
-  height: 36px;
-  width: 100px;
-  cursor: pointer;
-  background-color: #e0e1e2;
-  color: #00000099 !important;
-  font-weight: 700;
-  border-radius: 0.28571429rem;
+const FileButton = styled('label')`
```
Comment (hedvigoscar) [inline src/components/chat/insurance-table-rows/InsuranceTableRows.tsx]:
```suggestion
const FileButton = styled('label')({
```

Comment (artemfastdev) [reply src/components/chat/insurance-table-rows/InsuranceTableRows.tsx]: done

File: src/components/chat/insurance-table-rows/InsuranceTableRows.tsx [hunk 2]
```diff
@@ -3,44 +3,48 @@ import { ACTIVATION_DATE, CANCELLATION_DATE } from 'lib/messageTypes'
 import * as moment from 'moment'
 import * as PropTypes from 'prop-types'
 import * as React from 'react'
+import styled, { css } from 'react-emotion'
```
Comment (hedvigoscar) [inline src/components/chat/insurance-table-rows/InsuranceTableRows.tsx]:
```suggestion
import styled from 'react-emotion'
```

import appears to be unused?

Comment (artemfastdev) [reply src/components/chat/insurance-table-rows/InsuranceTableRows.tsx]: yep, done

**Pull request merged**